onebusaway-transit-data/src/main/java/org/onebusaway/transit_data/model/ArrivalsAndDeparturesQueryBean.java

```diff
@@ -49,6 +49,8 @@ public final class ArrivalsAndDeparturesQueryBean implements Serializable {
   private FilterChain systemFilterChain = new FilterChain();
   private FilterChain instanceFilterChain = new FilterChain();
 
+  private RouteSort subwayRouteSort = new RouteSort("1,2,3,4,5,6,7,A,C,E,B,D,F,M,G,L,N,Q,R,W,J,Z,Sf,S,Sr,SIR");
```

Comment:
I'm not comfortable with this code here.  Can you remove it to configuration, the default will instead be an alphanumeric sort?